README.md

```diff
@@ -26,6 +26,17 @@ The `<script>` block uses the syntax highlighting currently active for you norma
 
 2. Explicitly disable Sublime's default `JavaScript` package. This allows the Babel package to be applied the embedded `<script>` in `*.vue` files. You may need to restart Sublime for this to take effect.
 
+you can also include the lang attribute inside the `<script>` tag. 
```

Comment:
This should be the general recommendation (instead of suggesting to disable a default package) as it will be more portable. The part above should either be removed entirely or only mentioned as an alternative if `lang="jsx"` doesn't work.

Reply:
Thanks for the adjustments. You may resolve this (because I cannot).